### PR TITLE
Fix 'configure' command to handle configs with whitespaces

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Part of fixing of #83.

### CODE REVIEW

File: src/main/git-elegant-configure [hunk 1]
```diff
@@ -30,7 +30,7 @@ _configure() {
             read answer
             answer=${answer:-$default}
             if [ -n "${answer}" ]; then
-                git config $MODE $(eval "echo -n \$${f}_key") $answer
+                git config $MODE $(eval "echo -n \$${f}_key") "$answer"
```
Comment (extsoft) [inline src/main/git-elegant-configure]: Please add a unit test.

Comment (AntonKarabaza) [reply src/main/git-elegant-configure]: @extsoft But we actually mock `git config $MODE $(eval "echo -n \$${f}_key") "$answer"` command during unit testing. So there will be no difference whether I mock it like:
`fake-pass git "config --global user.name UserName"`
or
`fake-pass git "config --global user.name User Name"`

Comment (extsoft) [reply src/main/git-elegant-configure]: @AntonKarabaza Then let's wait for #89.

**Comment by extsoft** [comment]
@AntonKarabaza please update this branch with the latest version.

**Comment by AntonKarabaza** [comment]
@extsoft Done. Branch rebased.

**Comment by extsoft** [comment]
@rultor please merge.

**Pull request merged**